README.md

```diff
@@ -88,12 +88,153 @@ kubectl patch clusteradmissionpolicy psp-capabilities -p '{"metadata":{"finalize
 The [documentation](https://docs.kubewarden.io) provides more insights
 about how the project works and how to use it.
 
-# Software bill of materials
+# Software bill of materials & provenance
+
+Kubewarden controller has its software bill of materials (SBOM) and build
+[Provenance](https://slsa.dev/spec/v1.0/provenance) information published every
+release. It follows the [SPDX](https://spdx.dev/) format and
+[SLSA](https://slsa.dev/provenance/v0.2#schema) provenance schema respectfully.
+Both of the files are generated by [Docker
+buildx](https://docs.docker.com/build/metadata/attestations/) during the build
+process and stored in the container registry together with the container image
+as well as upload in the release page.
+
+You can find them together with the signature and certificate used to sign it
+in the [release
+assets](https://github.com/kubewarden/kubewarden-controller/releases), and
+attached to the image as JSON-encoded documents following the [in-toto SPDX
+predicate](https://github.com/in-toto/attestation/blob/main/spec/predicates/spdx.md)
+format. You can obtain them with
+[`crane`](https://github.com/google/go-containerregistry/blob/main/cmd/crane/README.md)
+or [`docker buildx imagetools
+inspect`](https://docs.docker.com/reference/cli/docker/buildx/imagetools/inspect).
+
+You can verify the container image with:
 
-Kubewarden controller has its software bill of materials (SBOM) published every
-release. It follows the [SPDX](https://spdx.dev/) version 2.2 format and you can
-find it together with the signature and certificate used to sign it in the
-[release assets](https://github.com/kubewarden/kubewarden-controller/releases)
+```shell
+cosign verify-blob --certificate-oidc-issuer=https://token.actions.githubusercontent.com  \
+    --certificate-identity-regexp="https://github.com/kubewarden/kubewarden-controller*"  \
```

Comment:
This example needs to be fully qualified, otherwise users may be checking against the wrong identity.